package.json

```diff
@@ -21,5 +22,10 @@
   "bugs": {
     "url": "https://github.com/RDesukevich/if-js/issues"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": ""
+    }
+  },
```

Comment:
you need to add some checks for code, f.e. I would recommend watch lint-staged library